.github/actions/setup-sentry/action.yml

```diff
@@ -0,0 +1,143 @@
+name: 'Sentry Setup'
+description: 'Sets up a Sentry test environment'
+inputs:
+  snuba:
+    description: 'Is snuba required?'
+    required: false
+    default: 'false'
+  kafka:
+    description: 'Is kafka required?'
+    required: false
+    default: 'false'
+  chartcuterie:
```

Comment:
I forked the action and tried to minimize the number of changes, thus, there's code that is not necessary for Relay integration tests.